### PR TITLE
Ungalaxyification

### PR DESCRIPTION
**Pull request by natefoo**
Make this a more generic role that does not automatically configure Galaxy CVMFS repos by default (but retain a single config option to enable them) to make this more suitable for uploading to Ansible Galaxy for general consumption.

### CODE REVIEW

**Comment by afgane** [comment]
LGTM, perhaps just a bit of documentation in the README for general consumption as well?

**Comment by erasche** [comment]
Maybe we could consider hiding the galaxy values behind a flag of some sort? (I don't want to have to diff my variables against some text in the readme or a file every time I pull in a newer version of this module.)

**Comment by natefoo** [comment]
@afgane That's the plan, it's the main reason it's still WIP. That and it's untested.

@erasche Not sure I follow. This puts them behind `galaxy_cvmfs_repos_enabled`.

**Comment by erasche** [comment]
@natefoo I'm blind. I missed that, sorry. I guess I saw the deletion of the old servers variable and didn't remember that that variable wasn't being consumed anyway and I'd already filed an issue about that.

**Comment by erasche** [comment]
Fixes https://github.com/galaxyproject/ansible-cvmfs/issues/3

File: defaults/main.yml [hunk 1]
```diff
@@ -87,28 +66,7 @@ galaxy_cvmfs_keys:
...
-      AwEAATANBgkqhkiG9w0BAQsFAAOCAQEAW0lijt9nXWIFX+ZiS6GqowD0XNzoWwwa
-      05YHhivKlFzadBpsO9W5/Evq5Upi6plLQw0m2gYl6zaZKxXB5N90nvKqSL8tp4AZ
-      C+qJdwBSJjG+rWZLh71AgTAfbKFxQUQea4s1yq6Y/iM/KpQj3xvGWCTtPHNVFqgX
-      yc7Z5XE9YQIVsKO1Z4A1SVyp/F+xn9ikRfE1USuVXd14icPgSkLd4uCF1is9Ns2h
-      IVsc8y8pyu4e2uiuvAiDkIWDwTj5LwvQlzCeAn4Z2crBEMDXOg5Ax7DE+8K+VaS7
-      umt/c7usl22IscRFArkCPbTMJuUxyJYH8g9rl1AiziVw9zC9SCbD2w==
-      -----END CERTIFICATE-----
-  - path: /etc/cvmfs/keys/main.galaxyproject.org.pub
+  - path: /etc/cvmfs/keys/galaxyproject.org/main.galaxyproject.org.pub
     owner: g2main
```
[the remark this answers is not in the document]
Comment (natefoo) [reply defaults/main.yml]: Oh yeah, good catch. This is just done to replicate what happens when you do `cvmfs_server mkfs` or `cvmfs_server add-replica` with the "owner" flag set.

**Comment by natefoo** [comment]
The last few changes are untested but I think this is nearly ready.

**Comment by erasche** [comment]
Tested on a machine, client role works fine

**Pull request merged**